src/Lorentz/Contracts/BaseDAO/Doc.hs

```diff
@@ -0,0 +1,161 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Lorentz.Contracts.BaseDAO.Doc
+   ( callFA2Doc
+   , transferDoc
+   , balanceOfDoc
+   , tokenMetadataRegistryDoc
+   , updateOperatorsDoc
+
+   , transferOwnershipDoc
+   , acceptOwnershipDoc
+   , migrateDoc
+   , confirmMigrationDoc
+
+   , proposeDoc
+   , voteDoc
+   , setVotingPeriodDoc
+   , setQuorumThresholdDoc
+   , flushDoc
+
+   , burnDoc
+   , mintDoc
+   , transferContractTokensDoc
+   , tokenAddressDoc
+   ) where
+
+import Lorentz
+
+import Util.Markdown
+
+callFA2Doc :: Markdown
+callFA2Doc = [md|
+  Entrypoint to be called if you want to use one of FA2 entrypoints.
+  |]
+
+transferDoc :: Markdown
+transferDoc = [md|
+  Transfer tokens between a given account and each account from the given list.
+
+  It serves multiple purposes:
+  * If transaction `"from"` address equals to the admin address presented in storage, than  it is allowed for
+  any address and both `frozen` and `unfrozen` tokens
+  * Otherwise, it is allowed to transfer money only to the accounts presented as `"from"` operators in storage.
```

Comment:
I don't understand this sentence. It's allowed to transfer money to any account. The restriction applies to `from` addresses: each address must be `SENDER` or have `SENDER` as an operator.